termlink/common.py

```diff
@@ -18,10 +18,28 @@
 
 from termlink.models import Coding, Relationship, RelationshipSchema
 
+_SCOPE_TO_INVERSE_SCOPE = {
+    'equivalent_to': 'equivalent_to'
+}
+
 _SCOPE_TO_EQUIVALENCE = {
     'equivalent_to': 'equivalent'
 }
 
+def _to_inverse_scope(scope):
+    """Converts a scope into it's inverse scope
```

Comment:
I'm pretty sure you meant "its" here (possessive form) rather than "it's". I know how you detest grammar mistakes!

Reply:
Lol, thanks!